tests/unit_tests/test_dqc_us_0004.py

```diff
@@ -36,12 +40,12 @@ def test_bv_errors(self, patched_decimals):
             liabilities_concept.qname: [liabilities_fact]
         }
 
-        modelXbrl = Mock()
-        modelXbrl.nameConcepts = mock_name_concepts_dict
-        modelXbrl.factsByQname = mock_facts_by_qname
+        model_xbrl = Mock(spec='arelle.modelXbrl.ModelXbrl')
```

Comment:
Actually I looked into this a bit, and you are right. Technically you can pass in a list of strings to define a spec, but it is better to just pass in the object. So in these cases we should just import the relent object and pass it in here. I was thinking of the patch `autospec=True`, mah bad.